docs/source/About OpenReview.rst

```diff
@@ -0,0 +1,38 @@
+Read more about OpenReview
```

Comment:
can the name of this file be changed to something without spaces?

Reply:
Done. Changed name of "About OpenReview.rst" to "about.rst" and "autodoc.rst" to "api.rst".